atomic_reactor/utils/cachito.py

```diff
@@ -147,14 +149,22 @@ def wait_for_request(
 
             # All other states are expected to be transient and are not checked.
 
-            elapsed = time.time() - start_time
+            # If "last_updated_value" does not match the "updated" value of the
+            # request from Cachito, then we know Cachito has performed some work
+            # since the last check, so the timer resets.
+            if last_updated_value is None or last_updated_value != response_json['updated']:
```

Comment:
I added `last_updated_value is None` to the if statement in the event Cachito has a bug and returns a null `updated` value. Let me know if you think this is too defensive and I can remove it.

Reply:
You could simply do:
```
last_update_time = response_json['updated'] or last_update_time or time.time()
```

However, in either case, if there's a Cachito bug this could run indefinitely.

Reply:
`last_update_time` is float of seconds but `response_json['updated']` is a string, so this won't work.